_source/logzio_collections/_prometheus-sources/telegraf-zookeeper.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Ship Apache Zookeper metrics via Telegraf
+logo:
+  logofile: zookeper-telegraf.png
+  orientation: vertical
+data-source: Apache Zookeper metrics over Telegraf
+templates: ["docker"]
+contributors:
+  - daniel-tk
+  - nshishkin
+shipping-tags:  
+  - prometheus
+order: 800
+---
+
+## Overview
+
+Telegraf is a plug-in driven server agent for collecting and sending metrics and events from databases, systems and IoT sensors.
+
+To send your Prometheus-format Apache Zookeper metrics to Logz.io, you need to add the **inputs.zookeper** and **outputs.http** plug-ins to your Telegraf configuration file.
+
+#### Configuring Telegraf to send your metrics data to Logz.io
+
+<div class="tasklist">
+
+##### Set up Telegraf v1.17 or higher
+
+{% include metric-shipping/telegraf-setup.md %}
+
+##### Add the inputs.zookeper plug-in
+
+First you need to configure the input plug-in to enable Telegraf to scrape the Apache Zookeper data from your hosts. To do this, add the following code to the configuration file:
+
```

Comment:
need a sentence to add under servers the ip / hostname with port of the zookeeper servers